<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataEntryDialog</class>
 <widget class="QDialog" name="DataEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>صفحة ادخال البيانات</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5">
    <widget class="QLineEdit" name="IDLineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="backButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>الرجوع</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="enterButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>ادخل البيانات</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>الكود</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="providerNameLineEdit"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>اسم المنتج</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="priceLineEdit"/>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>اسم المورد</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>السعر بالجنية</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QTextEdit" name="productNameLineEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionEnter">
   <property name="text">
    <string>enter</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>back</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>enterButton</sender>
   <signal>clicked()</signal>
   <receiver>actionEnter</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>actionBack</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
